pkgs/applications/misc/1password/default.nix

```diff
@@ -24,6 +24,19 @@ stdenv.mkDerivation rec {
       }
     else throw "Architecture not supported";
 
+  nativeBuildInputs = [ verifySignatureHook ];
+
+  signaturePublicKey = fetchpgpkey {
+    url = https://keybase.io/1password/pgp_keys.asc;
```

Comment:
What if they update their key?

Reply:
Can we fetch their key from a key-server using the current keyid instead?

Reply:
Keyservers are just one of many ways of fetching OpenPGP keys, I don't think it would make sense to force downloads through them (also, we'd want to fetch using the fingerprint, not the keyid).

Having a `fetchFromSks` wrapper of `fetchpgpkey` that fetches from the sks-keyservers network might be a meaningful addition, but I don't think it's required for the first iteration